src/components/EditItemsList/EditItemsList.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { Wrap, Input, Box, Button, Flex } from '@chakra-ui/react';
+import ItemCard from '../ItemCard/ItemCard';
+
+const EditItemsList = ({ items, setItems, isAccepted = false }) => {
+  const ref = useRef();
+
+  const handleAdd = () => {
+    // add validation to make sure current value isn't already in items
+    // if it is already in items, set an error message (see chakra form control)
+    setItems(prev => [...prev, { name: ref.current.value, accepted: isAccepted }]);
+  };
+  // maxW prop
+  return (
+    <Box bg="gray.50" width="50%" height="100%" mx="3vh" overflow="hidden">
+      <Wrap height="100%" minHeight="50vh" my="3%">
+        {items
+          .filter(({ accepted }) => accepted === isAccepted)
```

Comment:
🚫 **[eslint]** <[react/prop-types](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/prop-types.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'items.filter' is missing in props validation